launchlib/launcher.go

```diff
@@ -376,3 +418,12 @@ func isMaxRAMPercentage(arg string) bool {
 func isInitialRAMPercentage(arg string) bool {
 	return strings.HasPrefix(arg, "-XX:InitialRAMPercentage=")
 }
+
+// ComputeJVMHeapSizeInBytes If the experimental `ContainerV2` is set, compute the heap size to be 75% of
+// the heap minus 3mb per processor, with a minimum value of 50% of the heap.
+func ComputeJVMHeapSizeInBytes(hostProcessorCount int, cgroupMemoryLimitInBytes uint64) uint64 {
+	var heapLimit = float64(cgroupMemoryLimitInBytes)
```

Comment:
terminology nit: Let's call this the memory limit, we'll never be able to launch a jvm with a heap very close to this value